ui/src/ScholarReader.tsx

```diff
@@ -430,14 +417,18 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
     const SCROLL_OFFSET_X = -400;
     const SCROLL_OFFSET_Y = +100;
 
-    if (this.state.pdfViewer !== null) {
+    if (this.state.pdfViewer !== null && this.state.pages !== null && this.state.pages[box.page + 1] !== undefined) {
+      const { left, top } = selectors.divDimensionStyles(
+        this.state.pages[box.page + 1].view,
+        box 
+      )
```

Comment:
I'm not sure this is quite right. If I recall correctly (though I might be wrong, perhaps you can double check?), `scrollPageIntoView` expects the scroll coordinates in PDF coordinates. `divDimensionStyles` returns pixel coordinates, which will change based on the size of the page div (which will change with zooming, etc.).

I think this might need to be updated to convert `left` and `top` to PDF coordinates. I wonder if the PDF page width and height is available as metadata on the `view` object.

Reply:
Aaahhh I was wondering why it wasn't working hahaha, thanks! I'll look into switching this over after we talk if we decide to use it :). If not I'll return it to its original state